Expecto/Expecto.fs

```diff
@@ -485,7 +495,7 @@ module Impl =
                   eventX "Maximum memory usage was {memory} KB and exceeded the limit set at {limit} KB.\nRunning tests:\n\t{timeout}"
                   >> setField "memory" (summary.maxMemory / 1024L |> int |> commaString)
                   >> setField "limit" (summary.memoryLimit / 1024L |> int |> commaString)
-                  >> setField "timeout" (summary.timedOut |> Seq.map (fun t -> t.name) |> String.concat "\n\t"))
+                  >> setField "timeout" (summary.timedOut |> Seq.map (fun t -> getName t.name) |> String.concat "\n\t"))
```

Comment:
Can't we put this on the type instead? I.e. `t` could have properties that do some of the transformations

Reply:
Done